packages/gatsby-theme-docs/src/components/card.js

```diff
@@ -1,102 +1,30 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import styled from '@emotion/styled';
-import { css } from '@emotion/react';
-import { designSystem, Markdown } from '@commercetools-docs/ui-kit';
-import markdownFragmentToReact from '../utils/markdown-fragment-to-react';
-import Link from './link';
+import {
+  Markdown,
+  markdownFragmentToReact,
+  cardElements,
```

Comment:
I did not expose all the card component pieces individually becasue a) they don't make sense and are not usable standalone b) they would clutter the namespace too much.